Riot/ViewController/RoomViewController.m

```diff
@@ -1605,7 +1603,33 @@ - (void)refreshPreviewHeader:(BOOL)isLandscapeOriented
         {
             previewHeader.mainHeaderContainer.hidden = NO;
             previewHeader.mainHeaderBackgroundHeightConstraint.constant = previewHeader.mainHeaderContainer.frame.size.height;
-            
+
+            if ([previewHeader isKindOfClass:PreviewRoomTitleView.class])
+            {
+                // In case of preview, update the header height so that we can
+                // display as much as possible the room topic in this header.
+                // Note: the header height is the previewHeader.mainHeaderBackgroundHeightConstraint
```

Comment:
I would say "Note: the header height is handled by the previewHeader.mainHeaderBackgroundHeightConstraint."
by removing the "just computed" too.